### ISSUE TITLE
Microscopy: image loading indicator

### ISSUE THREAD

**Issue by pdpinch** [starteam/starcellbio_html#301]
As a user, it's hard to tell when a microscopy image has loaded, or if it's failed.

**Comment by ald32** [comment]
Yes, I spoke to Anna about this last week. I think we need to add in an indicator to tell the users that the image has loaded.

**Comment by pdpinch** [comment]
We could change the default color of the microscope background, but it would still need to be updated to black when the slide is uploaded. So, at the end of the day we'll need to set up some kind of onloaded event for the images.

**Comment by annagav** [comment]
Need to find instances where the image fails to load.
This issue is very old. With the AB this problem might not be relevant.
